Purchasing.Mvc/Views/Order/_ReviewLineItems.cshtml

```diff
@@ -91,7 +91,16 @@
                                             @foreach (var split in Model.Splits.Where(x => x.LineItem == line))
                                             {  
                                                 <tr>
-                                                    <td>@(string.IsNullOrWhiteSpace(split.Account) && string.IsNullOrWhiteSpace(split.FinancialSegmentString) ? "n/a" : split.AccountDisplay)</td>
+                                                    @if (!string.IsNullOrWhiteSpace(split.FinancialSegmentString))
+                                                    {
+                                                        var link = $"https://finjector.ucdavis.edu/Details/{split.FinancialSegmentString}";
+                                                        <td><a href=@link target="_blank" style="text-decoration:underline">@split.AccountDisplay</a> </td>
+                                                    }
+                                                    else
+                                                    {
+                                                        <td>@(string.IsNullOrWhiteSpace(split.Account) && string.IsNullOrWhiteSpace(split.FinancialSegmentString) ? "n/a" : split.AccountDisplay)</td>
```

Comment:
This `string.IsNullOrWhiteSpace(split.FinancialSegmentString)` seems unnecessary

Reply:
Yeah, I'm just lazy :)